packer/variables.pkr.hcl

```diff
@@ -15,7 +15,7 @@ variable "ubuntu_version" {
     minor_version = string
   })
   default = {
-    version       = "22.04"
+    version       = "22.04.1"
     minor_version = ""
```

Comment:
Hmm I had sorta expected we'd use the `minor_version` for this. But honestly I didn't realize that this would work too!